### ISSUE TITLE
Make Cosmic Python more discoverable to the Python/Event Sourcing/CQRS community

### ISSUE THREAD

**Issue by stevesimmons** [cosmicpython/book#193]
Hi guys,

Have you thought about doing some SEO-friendly blog posts to make your in-progress work more visible?

My recent Google searches for 'python cqrs' and 'python event sourcing' never hinted that you were working on this book. I found it accidentally via an linked issue on the SQLAlchemy issue tracker.

My personal motivation is my previous job was at a bank that used an Python event-sourced trading system. I really like the event-sourced approach, and recently started prototyping stuff at home. I didn't find any mature open-source Python event store projects, and ended up using the Postgres message queue parts of Scott Bellware's Ruby message_db event store (https://github.com/message-db/message-db). 

Thank you
Stephen

**Comment by hjwp** [comment]
yes!  an eventsourcing blog post is definitely on the cards.  maybe even an extra appendix in a future edition.

in the meantime you may enjoy some bob content on the topic here:
https://www.youtube.com/watch?v=0l8vuYaaBUs
https://github.com/bobthemighty/eventsourcing-101
